src/installCommand/index.js

```diff
@@ -108,9 +109,46 @@ const installationSpec = {
         await applyPatch(packageJson, target);
       }
     },
+
+    installDidComplete: async (installedPackages) => {
+      if (!installedPackages) {
+        return;
+      }
+      await linkConditionalDependencies(installedPackages);
```

Comment:
I could imagine an API like:

```
installWillLink  : tentativeListOfLinking => augmentedListOfLinking
```

Where you get the "tentative list" passed as an arg, and then you can return an augmented/reduced version of that list to help influence exactly what is linked.